### ISSUE TITLE
Package Control causes plugin_host to crash occasionally 

### ISSUE THREAD

**Issue by mtman** [wbond/package_control#873]
I'm certain that crashes are caused by Package Control as I tested the same config & plugins, only with Package Control removed, for at least 1 month and I had not encountered a single crash. I put it back, updated, and crashes has started occurring again.

Details: https://www.sublimetext.com/forum/viewtopic.php?f=3&t=13024

I'm running ST3 on Win7 x64.

**Comment by wbond** [comment]
Is `sublime_text.exe` crashing, or `plugin_host.exe`?

If it is `plugin_host.exe`, please provide a debug log.

**Comment by mtman** [comment]
Only plugin_host.exe.

Console output:
`error: plugin_host has exited unexpectedly, plugin functionality won't be available until Sublime Text has been restarted`

Dmp file: https://db.tt/RLOjeYAZ

**Comment by wbond** [comment]
For the debug log, please turn on the [debug setting](https://packagecontrol.io/docs/settings#setting-debug) and then provide the full console output.

**Comment by mtman** [comment]
```
DPI scale: 1
startup, version: 3065 windows x32 channel: stable
executable: /C/Program Files (x86)/Sublime Text 3/sublime_text.exe
working dir: /C/Program Files (x86)/Sublime Text 3
packages path: /C/Users/mtman/AppData/Roaming/Sublime Text 3/Packages
state path: /C/Users/mtman/AppData/Roaming/Sublime Text 3/Local
hardware concurrency: 2
zip path: /C/Program Files (x86)/Sublime Text 3/Packages
zip path: /C/Users/mtman/AppData/Roaming/Sublime Text 3/Installed Packages
found 22 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 52 files for base name Main.sublime-menu
loading bindings
loading pointer bindings
found 1 files for base name Afterglow-green.sublime-theme
theme loaded
app ready
wrote startup cache, added files: 6 orphaned files: 0 total files: 337 cache hits: 331
pre session restore time: 2.95221
startup time: 3.31521
first paint time: 3.31921
launching: /C/Program Files (x86)/Sublime Text 3/plugin_host.exe
unknown include #documentation
unknown include #documentation
unknown include #documentation
unknown include #documentation
unknown include #documentation
unknown include #documentation
unknown include #documentation
unknown include #documentation
unknown include #documentation
loaded 1077 snippets
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin 0_package_control_loader.00-package_control
reloading plugin 0_package_control_loader.02-bz2
reloading plugin AngularJS.AngularJS-sublime-package
reloading plugin AngularJS.jscompletions
reloading plugin AngularJS.message
reloading plugin AngularJS.sublime_commands
reloading plugin AngularJS.viewlocation
reloading plugin AutoHotkey.ahkexec
reloading plugin Better TypeScript.TypeScript
reloading plugin ColorPicker.sublimecp
reloading plugin Crypto.Crypto
reloading plugin Gitignore.gitignore
reloading plugin HTML5.encode_html_entities
reloading plugin HTML5.html_completions
reloading plugin LESS.less_completions
reloading plugin Less2Css.less2css
reloading plugin Less2Css.lesscompiler
reloading plugin MarkdownEditing.bootstrap
reloading plugin MarkdownEditing.custom_find_under_expand
reloading plugin MarkdownEditing.distraction_free_mode
reloading plugin MarkdownEditing.footnotes
reloading plugin MarkdownEditing.gather_missing_links
reloading plugin MarkdownEditing.indent_list_item
reloading plugin MarkdownEditing.indent_list_multiitem
reloading plugin MarkdownEditing.insert_references
reloading plugin MarkdownEditing.jumpToMarker
reloading plugin MarkdownEditing.lint
reloading plugin MarkdownEditing.list_markdown_references
reloading plugin MarkdownEditing.numbered_list
reloading plugin MarkdownEditing.paste_as_link
reloading plugin MarkdownEditing.paste_as_reference
reloading plugin MarkdownEditing.quote_indenting
reloading plugin MarkdownEditing.underlined_headers
reloading plugin Package Control.bootstrap
reloading plugin Package Control.Package Control
reloading plugin Pretty JSON.PrettyJson
[WinError 2] The system cannot find the file specified
reloading plugin Tag.Edit
reloading plugin Tag.Tag
reloading plugin Tag.tag_classes
reloading plugin Tag.tag_close_tag
reloading plugin Tag.tag_close_tag_on_slash
reloading plugin Tag.tag_indent
reloading plugin Tag.tag_insert_as_tag
reloading plugin Tag.tag_lint
reloading plugin Tag.tag_remove
reloading plugin Tag.tag_remove_attributes
reloading plugin URLEncode.urlencode
reloading plugin CodeFormatter.CodeFormatter
CodeFormatter: formatting python files on ST3 not supported.
reloading plugin Colorsublime-Plugin-master.colorsublime-plugin
reloading plugin Gutter Color.file
reloading plugin Gutter Color.gutter_color
reloading plugin Gutter Color.line
reloading plugin OmniSharp.OmniSharpSublime
reloading plugin SFTP.SFTP
reloading plugin SublimeFileDiffs.file_diffs
reloading plugin SublimeREPL.__init__
reloading plugin SublimeREPL.completions
reloading plugin SublimeREPL.lang_integration
reloading plugin SublimeREPL.run_existing_command
reloading plugin SublimeREPL.sublimerepl
reloading plugin SublimeREPL.sublimerepl_build_system_hack
reloading plugin SublimeREPL.text_transfer
plugins loaded
CodeFormatter: Plugin Initialized
omnisharp plugin_loaded
Package Control: Fetching list of available packages
  Platform: windows-x32
  Sublime Text Version: 3065
  Package Control Version: 3.0.1
Package Control: Download Debug
  URL: https://packagecontrol.io/channel_v3.json
  Resolved IP: 50.116.33.29
  Timeout: 30
Package Control: WinINet Debug Proxy
  proxy: 
  proxy bypass: 
  proxy username: 
  proxy password: 
Package Control: WinINet HTTPS Debug General
  Server SSL Certificate:
    subject: US, Massachusetts, Private Organization, 471714639, US, Massachusetts, Newbury, Codex Non Sufficit LC, codexns.io
    issuer: US, GeoTrust Inc., GeoTrust EV SSL CA - G4
    common name: codexns.io
    issue date: Wed, 17 Dec 2014 01:00:00 GMT
    expire date: Fri, 30 Oct 2015 00:59:59 GMT
Package Control: WinINet HTTPS Debug Write
  GET /channel_v3.json HTTP/1.1
  If-Modified-Since: Fri, 27 Feb 2015 16:50:04 GMT
  If-None-Match: "54f0a03c-2f978"
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.0.1
  Host: packagecontrol.io
  Connection: Keep-Alive
  Cache-Control: no-cache
Package Control: WinINet HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Mon, 02 Mar 2015 10:24:55 GMT
  Content-Type: application/json
  Content-Length: 197433
  Last-Modified: Mon, 02 Mar 2015 10:20:03 GMT
  Connection: keep-alive
  ETag: "54f43953-30339"
  Content-Encoding: bzip2
  Vary: Accept-Encoding
  Strict-Transport-Security: max-age=15552000
  Accept-Ranges: bytes
Package Control: Caching https://packagecontrol.io/channel_v3.json in C:\Users\mtman\AppData\Roaming\Sublime Text 3\Packages\User\Package Control.cache\01524fae79697630d0454ba3fabd9414
Package Control: Fetching list of available packages
  Platform: windows-x32
  Sublime Text Version: 3065
  Package Control Version: 3.0.1
Package Control: WinINet HTTPS Debug General
  Closing connection to packagecontrol.io on port 443 after 1 request
Package Control: Installing 3 upgrades
ignored packages updated to: [Markdown, PlainTasks, Vintage, Markdown Preview, Seti_UI, Seti_UX]
found 1 files for base name Afterglow-green.sublime-theme
reloading Packages/Seti_UI/+res/Ext/CSS.sublime-settings
reloading Packages/Seti_UI/+res/Ext/HTML (Rails).sublime-settings
reloading Packages/Seti_UI/+res/Ext/HTML.sublime-settings
reloading Packages/Seti_UI/+res/Ext/LESS.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Plain text.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Python.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Ruby Haml.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Ruby.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Sass.sublime-settings
reloading Packages/Seti_UI/+res/Ext/SCSS.sublime-settings
generating syntax summary
generating meta info summary
reloading Packages/User/Package Control.sublime-settings
Package Control: Fetching list of available packages
  Platform: windows-x32
  Sublime Text Version: 3065
  Package Control Version: 3.0.1
theme loaded
Package Control: Download Debug
  URL: https://codeload.github.com/aziz/PlainTasks/zip/master
  Resolved IP: 192.30.252.146
  Timeout: 30
reloading Packages/User/Preferences.sublime-settings
found 1 files for base name Afterglow-green.sublime-theme
theme loaded
Package Control: Attempting to use Urllib downloader due to WinINet error: Error downloading package. Error checking for server certificate revocation (errno 12057) during HTTP write phase of downloading https://codeload.github.com/aziz/PlainTasks/zip/master.
Package Control: Download Debug
  URL: https://codeload.github.com/aziz/PlainTasks/zip/master
  Resolved IP: 192.30.252.144
  Timeout: 30
Package Control: Urllib Debug Proxy
  http_proxy: 
  https_proxy: 
  proxy_username: 
  proxy_password: 
Package Control: Generating new CA bundle from system certificate store
Package Control: Exported certificate "Microsoft Root Certificate Authority"
Package Control: Skipping certificate "Thawte Timestamping CA" since it uses the signature algorithm md5WithRSAEncryption
Package Control: Skipping certificate "Microsoft Root Authority" since it uses the signature algorithm md5WithRSAEncryption
Package Control: Exported certificate "Microsoft Root Certificate Authority 2011"
Package Control: Skipping certificate "Microsoft Authenticode(tm) Root Authority" since it is no longer valid
Package Control: Exported certificate "Microsoft Root Certificate Authority 2010"
Package Control: Skipping certificate "Copyright (c) 1997 Microsoft Corp." since it is no longer valid
Package Control: Skipping certificate "NO LIABILITY ACCEPTED, (c)97 VeriSign, Inc." since it is no longer valid
Package Control: Exported certificate "thawte Primary Root CA - G3"
Package Control: Exported certificate "UTN-USERFirst-Object"
Package Control: Exported certificate "Thawte Premium Server CA"
Package Control: Exported certificate "GeoTrust Global CA"
Package Control: Exported certificate "Baltimore CyberTrust Root"
Package Control: Exported certificate "Equifax Secure Certificate Authority"
Package Control: Exported certificate "Starfield Root Certificate Authority - G2"
Package Control: Exported certificate "GlobalSign Root CA"
Package Control: Exported certificate "Starfield Class 2 Certification Authority"
Package Control: Exported certificate "DigiCert Global Root CA"
Package Control: Skipping certificate "GTE CyberTrust Global Root" since it uses the signature algorithm md5WithRSAEncryption
Package Control: Exported certificate "thawte Primary Root CA"
Package Control: Skipping certificate "Class 3 Public Primary Certification Authority" since it uses the signature algorithm md2WithRSAEncryption
Package Control: Skipping certificate "Thawte Premium Server CA" since it uses the signature algorithm md5WithRSAEncryption
Package Control: Exported certificate "Certum CA"
Package Control: Exported certificate "DigiCert High Assurance EV Root CA"
Package Control: Exported certificate "Entrust.net Certification Authority (2048)"
Package Control: Skipping certificate "Class 3 Public Primary Certification Authority" since it is no longer valid
Package Control: Exported certificate "VeriSign Class 3 Public Primary Certification Authority - G5"
Package Control: Exported certificate "Go Daddy Root Certificate Authority - G2"
Package Control: Exported certificate "StartCom Certification Authority"
Package Control: Exported certificate "GeoTrust Primary Certification Authority"
Package Control: Exported certificate "http://www.valicert.com/"
Package Control: Exported certificate "Go Daddy Class 2 Certification Authority"
Package Control: Exported certificate "DigiCert Assured ID Root CA"
Package Control: Exported certificate "UTN-USERFirst-Hardware"
Package Control: Exported certificate "AddTrust External CA Root"
Package Control: Exported certificate "MSIT Machine Auth CA 2"
Package Control: Exported certificate "COMODO High-Assurance Secure Server CA"
Package Control: Exported certificate "Microsoft Internet Authority"
Package Control: Exported certificate "Microsoft IT SSL SHA2"
Package Control: Exported certificate "Microsoft Code Signing PCA"
Package Control: Skipping certificate "Root Agency" since it uses the signature algorithm md5WithRSAEncryption
Package Control: Exported certificate "www.verisign.com/CPS Incorp.by Ref. LIABILITY LTD.(c)97 VeriSign"
Package Control: Skipping certificate "Microsoft Windows Hardware Compatibility" since it is no longer valid
Package Control: Finished generating new CA bundle at C:\Users\mtman\AppData\Roaming\Sublime Text 3\Packages\User\Package Control.system-ca-bundle
Package Control: Regnerated the merged CA bundle from the system and user CA bundles
Package Control: Urllib HTTPS Debug General
  Connecting to codeload.github.com on port 443
Package Control: Urllib HTTPS Debug General
  Connecting to codeload.github.com on port 443
  CA certs file at C:\Users\mtman\AppData\Roaming\Sublime Text 3\Packages\User\Package Control.merged-ca-bundle
  Successfully upgraded connection to codeload.github.com:443 with SSL
  Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-SHA
  Server SSL certificate:
    subject: CN=*.github.com,O=GitHub, Inc.,L=San Francisco,S=California,C=US
    subject alt name: *.github.com, github.com
    expire date: Apr 12 12:00:00 2017 GMT
  Certificate validated for codeload.github.com
Package Control: Urllib HTTPS Debug Write
  GET /aziz/PlainTasks/zip/master HTTP/1.1
  User-Agent: Package Control v3.0.1
  Host: codeload.github.com
  Accept-Encoding: bzip2,gzip,deflate
  Connection: Keep-Alive
Package Control: Urllib HTTPS Debug Read
  HTTP/1.1 200 OK
  Content-Length: 265458
  Access-Control-Allow-Origin: https://render.githubusercontent.com
  Content-Security-Policy: default-src 'none'
  X-XSS-Protection: 1; mode=block
  X-Frame-Options: deny
  X-Content-Type-Options: nosniff
  Strict-Transport-Security: max-age=31536000
  Vary: Authorization,Accept-Encoding
  ETag: "18e1f9d61288621ec9be59dc01839d17ca8d30d5"
  Content-Type: application/zip
  Content-Disposition: attachment; filename=PlainTasks-master.zip
  Date: Mon, 02 Mar 2015 10:25:25 GMT
Package Control: Urllib HTTPS Debug General
  Closing connection to codeload.github.com on port 443 after 1 request
Package Control: Skipping cache since the response is a zip file
Package Control: Download Debug
  URL: https://packagecontrol.io/submit?package=PlainTasks&sublime_platform=windows&version=2015.02.28.03.15.45&old_version=2015.02.26.18.05.10&sublime_version=3065&package_control_version=3.0.1&operation=upgrade
  Resolved IP: 50.116.33.29
  Timeout: 30
Package Control: WinINet Debug Proxy
  proxy: 
  proxy bypass: 
  proxy username: 
  proxy password: 
Package Control: WinINet HTTPS Debug General
  Server SSL Certificate:
    subject: US, Massachusetts, Private Organization, 471714639, US, Massachusetts, Newbury, Codex Non Sufficit LC, codexns.io
    issuer: US, GeoTrust Inc., GeoTrust EV SSL CA - G4
    common name: codexns.io
    issue date: Wed, 17 Dec 2014 01:00:00 GMT
    expire date: Fri, 30 Oct 2015 00:59:59 GMT
Package Control: WinINet HTTPS Debug Write
  GET /submit?package=PlainTasks&sublime_platform=windows&version=2015.02.28.03.15.45&old_version=2015.02.26.18.05.10&sublime_version=3065&package_control_version=3.0.1&operation=upgrade HTTP/1.1
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.0.1
  Host: packagecontrol.io
  Connection: Keep-Alive
  Cache-Control: no-cache
Package Control: WinINet HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Mon, 02 Mar 2015 10:25:27 GMT
  Content-Type: application/json
  Transfer-Encoding: chunked
  Connection: keep-alive
  Vary: Accept-Encoding
  X-App-Version: 1.1.1
  Strict-Transport-Security: max-age=15552000
  Content-Encoding: gzip
Package Control: Upgraded PlainTasks to v2015.02.28.03.15.45
Package Control: Fetching list of available packages
  Platform: windows-x32
  Sublime Text Version: 3065
  Package Control Version: 3.0.1
Package Control: Download Debug
  URL: https://codeload.github.com/ctf0/Seti_ST3/zip/master
  Resolved IP: 192.30.252.144
  Timeout: 30
Package Control: Urllib HTTPS Debug General
  Connecting to codeload.github.com on port 443
Package Control: Urllib HTTPS Debug General
  Connecting to codeload.github.com on port 443
  CA certs file at C:\Users\mtman\AppData\Roaming\Sublime Text 3\Packages\User\Package Control.merged-ca-bundle
  Successfully upgraded connection to codeload.github.com:443 with SSL
  Using TLSv1/SSLv3 with cipher ECDHE-RSA-AES128-SHA
  Server SSL certificate:
    subject: CN=*.github.com,O=GitHub, Inc.,L=San Francisco,S=California,C=US
    subject alt name: *.github.com, github.com
    expire date: Apr 12 12:00:00 2017 GMT
  Certificate validated for codeload.github.com
Package Control: Urllib HTTPS Debug Write
  GET /ctf0/Seti_ST3/zip/master HTTP/1.1
  User-Agent: Package Control v3.0.1
  Host: codeload.github.com
  Accept-Encoding: bzip2,gzip,deflate
  Connection: Keep-Alive
Package Control: Urllib HTTPS Debug Read
  HTTP/1.1 200 OK
  Content-Length: 1580522
  Access-Control-Allow-Origin: https://render.githubusercontent.com
  Content-Security-Policy: default-src 'none'
  X-XSS-Protection: 1; mode=block
  X-Frame-Options: deny
  X-Content-Type-Options: nosniff
  Strict-Transport-Security: max-age=31536000
  Vary: Authorization,Accept-Encoding
  ETag: "190070d4faf843b0f76f4529d42d2f89e4259bb4"
  Content-Type: application/zip
  Content-Disposition: attachment; filename=Seti_ST3-master.zip
  Date: Mon, 02 Mar 2015 10:25:27 GMT
Package Control: Urllib HTTPS Debug General
  Closing connection to codeload.github.com on port 443 after 1 request
Package Control: Skipping cache since the response is a zip file
Package Control: WinINet HTTPS Debug General
  Closing connection to packagecontrol.io on port 443 after 1 request
Package Control: Download Debug
  URL: https://packagecontrol.io/submit?package=Seti_UI&sublime_platform=windows&version=2015.02.27.22.51.13&old_version=2015.02.22.13.29.45&sublime_version=3065&package_control_version=3.0.1&operation=upgrade
  Resolved IP: 50.116.33.29
  Timeout: 30
Package Control: WinINet Debug Proxy
  proxy: 
  proxy bypass: 
  proxy username: 
  proxy password: 
Package Control: WinINet HTTPS Debug General
  Server SSL Certificate:
    subject: US, Massachusetts, Private Organization, 471714639, US, Massachusetts, Newbury, Codex Non Sufficit LC, codexns.io
    issuer: US, GeoTrust Inc., GeoTrust EV SSL CA - G4
    common name: codexns.io
    issue date: Wed, 17 Dec 2014 01:00:00 GMT
    expire date: Fri, 30 Oct 2015 00:59:59 GMT
Package Control: WinINet HTTPS Debug Write
  GET /submit?package=Seti_UI&sublime_platform=windows&version=2015.02.27.22.51.13&old_version=2015.02.22.13.29.45&sublime_version=3065&package_control_version=3.0.1&operation=upgrade HTTP/1.1
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.0.1
  Host: packagecontrol.io
  Connection: Keep-Alive
  Cache-Control: no-cache
Package Control: WinINet HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Mon, 02 Mar 2015 10:25:38 GMT
  Content-Type: application/json
  Transfer-Encoding: chunked
  Connection: keep-alive
  Vary: Accept-Encoding
  X-App-Version: 1.1.1
  Strict-Transport-Security: max-age=15552000
  Content-Encoding: gzip
Package Control: Upgraded Seti_UI to v2015.02.27.22.51.13
Package Control: Fetching list of available packages
  Platform: windows-x32
  Sublime Text Version: 3065
  Package Control Version: 3.0.1
Package Control: Download Debug
  URL: https://codeload.github.com/ctf0/Seti_UX/zip/master
  Resolved IP: 192.30.252.144
  Timeout: 30
Package Control: WinINet Debug Proxy
  proxy: 
  proxy bypass: 
  proxy username: 
  proxy password: 
Package Control: WinINet HTTPS Debug General
  Server SSL Certificate:
    subject: US, California, San Francisco, "GitHub, Inc.", *.github.com
    issuer: US, DigiCert Inc, www.digicert.com, DigiCert SHA2 High Assurance Server CA
    common name: *.github.com
    issue date: Tue, 08 Apr 2014 02:00:00 GMT
    expire date: Wed, 12 Apr 2017 14:00:00 GMT
Package Control: WinINet HTTPS Debug Write
  GET /ctf0/Seti_UX/zip/master HTTP/1.1
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.0.1
  Host: codeload.github.com
  Connection: Keep-Alive
  Cache-Control: no-cache
Package Control: WinINet HTTPS Debug Read
  HTTP/1.1 200 OK
  Content-Length: 836144
  Access-Control-Allow-Origin: https://render.githubusercontent.com
  Content-Security-Policy: default-src 'none'
  X-XSS-Protection: 1; mode=block
  X-Frame-Options: deny
  X-Content-Type-Options: nosniff
  Strict-Transport-Security: max-age=31536000
  Vary: Authorization,Accept-Encoding
  ETag: "ab59b23be428ab99902a9ace41f3ba4baa7ccd04"
  Content-Type: application/zip
  Content-Disposition: attachment; filename=Seti_UX-master.zip
  Date: Mon, 02 Mar 2015 10:25:38 GMT
ignored packages updated to: [Markdown, PlainTasks, Seti_UX, Markdown Preview, Vintage]
found 1 files for base name Afterglow-green.sublime-theme
Package Control: Skipping cache since the response is a zip file
theme loaded
generating meta info summary
reloading Packages/Seti_UI/+res/Ext/CSS.sublime-settings
reloading Packages/Seti_UI/+res/Ext/HTML (Rails).sublime-settings
reloading Packages/Seti_UI/+res/Ext/HTML.sublime-settings
reloading Packages/Seti_UI/+res/Ext/LESS.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Plain text.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Python.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Ruby Haml.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Ruby.sublime-settings
reloading Packages/Seti_UI/+res/Ext/Sass.sublime-settings
reloading Packages/Seti_UI/+res/Ext/SCSS.sublime-settings
generating syntax summary
Package Control: Download Debug
  URL: https://packagecontrol.io/submit?package=Seti_UX&sublime_platform=windows&version=2015.02.28.05.08.08&old_version=2015.02.22.13.44.19&sublime_version=3065&package_control_version=3.0.1&operation=upgrade
  Resolved IP: 50.116.33.29
  Timeout: 30
Package Control: WinINet HTTPS Debug General
  Re-using connection to packagecontrol.io on port 443 for request #1
reloading Packages/User/Package Control.sublime-settings
reloading Packages/User/Preferences.sublime-settings
found 1 files for base name Afterglow-green.sublime-theme
theme loaded
Package Control: WinINet Debug Proxy
  proxy: 
  proxy bypass: 
  proxy username: 
  proxy password: 
Package Control: WinINet HTTPS Debug Write
  GET /submit?package=Seti_UX&sublime_platform=windows&version=2015.02.28.05.08.08&old_version=2015.02.22.13.44.19&sublime_version=3065&package_control_version=3.0.1&operation=upgrade HTTP/1.1
  Accept-Encoding: bzip2,gzip,deflate
  User-Agent: Package Control v3.0.1
  Host: packagecontrol.io
  Connection: Keep-Alive
  Cache-Control: no-cache
Package Control: WinINet HTTPS Debug Read
  HTTP/1.1 200 OK
  Server: nginx
  Date: Mon, 02 Mar 2015 10:25:41 GMT
  Content-Type: application/json
  Transfer-Encoding: chunked
  Connection: keep-alive
  Vary: Accept-Encoding
  X-App-Version: 1.1.1
  Strict-Transport-Security: max-age=15552000
  Content-Encoding: gzip
Package Control: Upgraded Seti_UX to v2015.02.28.05.08.08
ignored packages updated to: [Markdown, Vintage, PlainTasks, Markdown Preview]
found 1 files for base name Afterglow-green.sublime-theme
theme loaded
reloading Packages/User/Package Control.sublime-settings
reloading Packages/User/Preferences.sublime-settings
found 1 files for base name Afterglow-green.sublime-theme
theme loaded
reloading Packages/User/Preferences.sublime-settings
found 1 files for base name Afterglow-green.sublime-theme
theme loaded
Package Control: WinINet HTTPS Debug General
  Closing connection to packagecontrol.io on port 443 after 2 requests
Package Control: WinINet HTTPS Debug General
  Closing connection to codeload.github.com on port 443 after 1 request
error: plugin_host has exited unexpectedly, plugin functionality won't be available until Sublime Text has been restarted
```

**Comment by wbond** [comment]
Hmm, it is strange that `plugin_host` would crash after Package Control is done and cleaned up. (The `Closing connection` messages are printed a few seconds after the last use of of a connection to a server).

Unfortunately without you being able to pinpoint the issue, and with there being a number of other packages installed on your machine, I don't think there is much I can do here.

You could try the latest build of ST3, however. In the release notes for 3075 it mentions a fix on Windows for a plugin_host crash. http://www.sublimetext.com/3dev
